client/app/dbaas/logs/detail/streams/follow/streams-follow.html

```diff
@@ -25,9 +25,9 @@
             <oui-action-menu-item data-ng-if="!ctrl.isConnectionClosed()" text="{{ ::'logs_streams_follow_close_connection' | translate}}" on-click="ctrl.closeConnection()"></oui-action-menu-item>
             <oui-action-menu-item text="{{ ::'logs_streams_follow_download_client' | translate}}" href="https://github.com/ovh/ldp-tail/releases" external></oui-action-menu-item>
             <oui-action-menu-item data-ng-if="ctrl.isConnectionClosed()" text="{{ ::'logs_streams_follow_open_connection' | translate}}" on-click="ctrl.openConnection()"></oui-action-menu-item>
-            <oui-action-menu-item text="{{ ::'logs_streams_follow_test_gelf' | translate}}" on-click="ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.GELF)"></oui-action-menu-item>
-            <oui-action-menu-item text="{{ ::'logs_streams_follow_test_ltsv' | translate}}" on-click="ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.LTSV)"></oui-action-menu-item>
-            <oui-action-menu-item text="{{ ::'logs_streams_follow_test_rfc' | translate}}" on-click="ctrl.testFlow(ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.RFC5424))"></oui-action-menu-item>
+            <oui-action-menu-item data-ng-if="ctrl.stream.data.info.isEditable" text="{{ ::'logs_streams_follow_test_gelf' | translate}}" on-click="ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.GELF)"></oui-action-menu-item>
```

Comment:
Could you please turn

````html
<oui-action-menu-item data-ng-if="ctrl.stream.data.info.isEditable" text="{{ ::'logs_streams_follow_test_gelf' | translate}}" on-click="ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.GELF)"></oui-action-menu-item>
````

into

````html
<oui-action-menu-item
    data-ng-if="ctrl.stream.data.info.isEditable"
    data-on-click="ctrl.testFlow(ctrl.LogsStreamsFollowService.testTypeEnum.GELF)">
    <span data-translate="logs_streams_follow_test_gelf"></span>
</oui-action-menu-item>
````
Meaning:
- set the `text` attribute value in a `span`
- add a `data-` before the `on-click`
- have only one attribute per line

And do this for all the `oui-action-menu-item` in this file ? 

Thanks :)

Reply:
Fixed